vnext/JSI/Shared/ByteArrayBuffer.h

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include <memory>
+#include "jsi/jsi.h"
+
+namespace Microsoft::JSI {
+
+class ByteArrayBuffer final : public facebook::jsi::Buffer {
+ public:
+  ByteArrayBuffer(size_t bufferSize)
+      : m_size(bufferSize), m_data(std::make_unique<uint8_t[]>(bufferSize)) {}
```

Comment:
>, [](start = 26, length = 1)

(nit) create a separate line for initialization

Reply:
Clang format strikes again...

---
In reply to: [333114640](https://github.com/microsoft/react-native-windows/pull/3366#discussion_r333114640) [](ancestors = 333114640)